### PR TITLE
Slaughterfish fix

### PR DESCRIPTION
**Pull request by cc9cii**
Please review/test/comment/critique...

### CODE REVIEW

File: apps/openmw/mwworld/physicssystem.cpp [hunk 1]
```diff
@@ -574,7 +619,7 @@
             {
                 float waterlevel = -std::numeric_limits<float>::max();
                 const ESM::Cell *cell = iter->first.getCell()->getCell();
-                if(cell->hasWater())
+                if(cell->hasWater()) // should also check cell->mHasWaterLevelRecord
```
Comment (ghost) [inline apps/openmw/mwworld/physicssystem.cpp]: hasWater() is completely sufficient. mHasWaterLevelRecord is only used during the loading phase.

Comment (cc9cii) [inline apps/openmw/mwworld/physicssystem.cpp]: Ok, noted.  I'll remove the comment but will wait a little for others to review.

**Comment by cc9cii** [comment]
PLEASE NOTE: recompiling against the latest master appears broken, please wait till I investigate what is going on.  Could be my local setup since textures are missing.

UPDATE: sorry, it was just my installation issue only
UPDATE2: happening again in my system, I'll close this request until I really know what is going on
UPDATE3: all fixed now

**Comment by cc9cii** [comment]
I've reverted the variable names so that it would be easier to compare to the previous version of the files.

**Pull request merged**